src/applications/vaos/covid-19-vaccine/components/ConfirmationPageV2.jsx

```diff
@@ -87,10 +95,18 @@ function ConfirmationPageV2({
           className="far fa-calendar vads-u-margin-right--1"
         />
         <AddToCalendar
-          summary="VA Appointment"
-          description=""
+          summary={`Appointment at ${clinic?.serviceName}`}
```

Comment:
Should this appointment match the VA in person copy? Seems like it's using clinic name in place of facility name

Reply:
I think the VA in person copy doc says clinic name.

Reply:
It does, apparently I'm going crazy.